src/ucp/api/ucp.h

```diff
@@ -1664,6 +1702,21 @@ ucs_status_ptr_t ucp_ep_modify_nb(ucp_ep_h ep, const ucp_ep_params_t *params);
 ucs_status_ptr_t ucp_ep_close_nb(ucp_ep_h ep, unsigned mode);
 
 
+/**
+ * @ingroup UCP_WORKER
+ *
+ * @brief Reject @ref ucp_ep_h "endpoint" creation.
+ *
+ * This routine releases the @ref ucp_ep_address_h and notifies client if there
+ * was set @ref ucp_ep_params_t::err_handler with status @ref UCS_ERR_REJECTED.
+ *
+ * @param [in]  listener    Handle to the listener which created the address.
+ * @param [in]  ep_addr     Handle to the remote endpoint address to reject.
+ *
+ */
+void ucp_listener_reject(ucp_listener_h listener, ucp_ep_address_h ep_addr);
```

Comment:
"address" terminology implies for a stateless blob of data, but here we reject it.
maybe should change "address" terminology to "connection request"?